pkgs/by-name/go/gojo/package.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  versionCheckHook,
+  nix-update-script,
+}:
+
+buildGoModule rec {
+  pname = "gojo";
+  version = "0.3.2";
+
+  src = fetchFromGitHub {
+    owner = "itchyny";
+    repo = pname;
```

```suggestion
    repo = "gojo";
```

This is a bad practice in general to use pname everywhere. If someone overrides the pname, this will break the whole derivation.